chrome/install.js

```diff
@@ -115,6 +121,29 @@ function checkJavaCommon(callback) {
 	});
 }
 
+function checkMacJDK() {
+	var deferred = Zotero.Promise.defer();
```

Comment:
Should normally use `return new Zotero.Promise((resolve, reject) => {` instead of `defer()` [1], but after switching to `Zotero.Utilities.Internal.exec()` below this can just use `var checkMacJDK = Zotero.Promise.coroutine(function* () {` (or `async function checkMacJDK()`, really, since even without Babel here we only run on 52+ now).

[1] Among other things, `new Zotero.Promise` would automatically treat thrown errors as rejections. (The `reject` is only necessary within another anonymous function.) As currently written, if bash didn't exist below or there was another problem with the nsIProcess, this function would release Zalgo.